coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1124,16 +1124,19 @@ func (s *server) notifyWorkspaceManualBuildFailed(ctx context.Context, workspace
 	}
 
 	for _, templateAdmin := range templateAdmins {
+		labels := map[string]string{
+			"name":                     workspace.Name,
+			"template_name":            template.DisplayName,
+			"template_version_name":    templateVersion.Name,
+			"initiator":                build.InitiatorByUsername,
+			"workspace_owner_username": workspaceOwner.Username,
+			"workspace_build_number":   strconv.Itoa(int(build.BuildNumber)),
+		}
+		if template.DisplayName == "" {
```

Comment:
Nit: stylistically I'd prefer do this above the labels definition and assign the value once to either the default or the alternative value.

Reply:
+1

Reply:
Done